### ISSUE TITLE
Support for simple ungrouped consumers

### ISSUE THREAD

**Issue by jmschles** [silviucpp/erlkaf#66]
I'm looking at switching from `Elsa` to `erlkaf`. My use case is to consume entire Kafka topics to populate in-memory caches on application boot. This requires two things from my consumers:

1. They start from the beginning of the topic every time the application starts, rather than starting from a previously-set offset
2. They're not part of a consumer group, or at least not part of a common or previously-used one, so each node in the cluster can maintain its own independent cache

I think I could accomplish this with randomly-generated unique consumer groups, but that feels like a hack, so I'm hoping there's a better way. I didn't see one looking through the code but might've missed it.

**Comment by jmschles** [comment]
Thanks @silviucpp! If I'm understanding correctly, think that workaround could solve the offset storage issue, but wouldn't address the requirement that each node consumes every message in the topic.

I definitely understand where you're coming from about priorities, and appreciate your taking the time to reply!

**Comment by silviucpp** [comment]
@jmschles you are totally right ! I'm glad to merge any contribution to the project . erlkaf it's currently used into a solution that handles tens of millions of voice calls every single day. For caching we are using KeyDB which works great for us with such volume. If helps I also have the erlang driver https://github.com/silviucpp/eredis_pool we are using with keydb active-active replaication and multi master.
